engine/crates/engine-v2/schema/src/sources/introspection.rs

```diff
@@ -592,6 +600,16 @@ impl<'a> IntrospectionSchemaBuilder<'a> {
         }
     }
 
+    fn insert_field_argument(&mut self, field_id: FieldId, argument: InputValueId) {
```

Comment:
IMHO we should have a function `insert_all_field_arguments()` to make sure we do add everything at once. Here it's fine because introspection likely won't change. But it's easy to misuse this function

Reply:
true, that came to mind too but I didn't act on it. Let's enforce this.